docs/interface.rst

```diff
@@ -325,74 +325,50 @@ There is a WebSocket-based status API located at ``/v1/status``.
-
-- ``"synchronizing"``: ``true`` or ``false``. When ``true`` the
-  magic-folder daemon is uploading data to or downloading data from
-  Tahoe-LAFS.
-- ``"tahoe"``: a dict containing status information about the Tahoe-LAFS connection
-  - ``"connected"``: the number of storage-servers the client is connected to
-  - ``"desired"``: the number of storage-servers we want to connect to
-  - ``"happy"``: ``true`` if ``"connected"`` is greater than the client's configured "happy"
-- ``"folders"`` contains keys mapping the folder name to:
-  - ``"uploads"`` and ``"downloads"`` contain currently queued or active uploads (or downloads). Each ``dict`` in these lists contain:
-
-    - ``"relpath"``: the relative-path
-    - ``"queued-at"``: the Unix timestamp when this item was queued
-    - ``"started-at"``: the Unix timestamp when we started uploading (or downloading) this item. This key will not exist until we do start.
-
-  - ``"recent"`` contains a list up to 30 of the most-recently updated files. Each ``dict`` in this list contains:
-
-    - ``"relpath"``: the relative path of this item
-    - ``"modified"``: the Unix timestamp when the on-disk file was most-recently modified
-    - ``"last-updated"``: the Unix timestamp when this item's state was updated in the magic-folder
-    - ``"conflicted"``: a boolean indicating if there is a conflict for this relative path
-  - ``"scanner"`` contains information about the local changes scanner
-    - ``"last-scan"``: ``null`` if no scan is completed yet, or the timestamp of the last completion
-  - ``"poller"`` contains information about the remote changes poller
-    - ``"last-poll"``: ``null`` if no scan is completed yet, or the timestamp of the last completion
-
-Clients should be tolerant of keys in the state they don't understand.
-Unknown state keys should be ignored.
-Note that ``"modified"`` is when the local state for thie item changed while ``"last-updated"`` is to do with the filesystem modification time.
-For example, a file may have an on-disk modification time that is older than the last time we updated our state about it, especially one that came from another device.
+...where the ``events`` list contains some non-zero number of event messages.
+The first message, upon connect, will likely contain many events: enough to give a consistent view of the current state.
+Thereafter, most messages will include only a single event (although clients should handle any number).
+
+Every event has a ``"kind"`` key describing what sort of message it is.
+Events will contain other keys; clients should be tolerant of keys in the state they don't understand.
 
 The client doesn't send any messages to the server; it is an error to do so.
+
+The follow event kinds are understood (see ``status.py`` for more details on the sending side, and ``cli.py`` for an example of receiving them):
+
+- ``"scanner"``: has key ``last-scan`` which is a unix-timestamp saying when we last looked for local changes.
```

Comment:
In keeping in line with the suggestion above, consider `scan-finished` or `scan-completed` here.